Bottom: 790547ea371563fb00399e3f4b5690993e27081c
Top:    d5df17da3f79eeadf8246278a56f4042fa7fa6ef
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-06 23:27:15 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
